source/voice_control/expanding_assist.markdown

```
@@ -18,6 +18,6 @@ Assist [up and running](/voice_control/) in any of the available devices and con
 ## Some ideas to expand your Assist setup
 
 1. Add custom sentences or modify existing ones. Learn how in this [custom sentences tutorial](/voice_control/custom_sentences/).
-2. [Create a personality](voice_control/assist_create_open_ai_personality/) for Assist using AI
+2. [Create a personality](/voice_control/assist_create_open_ai_personality/) for Assist using AI
 3. Customize your [own wake words ](/voice_control/create_wake_word/) for Assist - only available if you have your own hardware setup like [ESP32-S3 BOX](/voice_control/s3_box_voice_assistant/) or [ATOM Echo](/voice_control/thirteen-usd-voice-remote/).
```

Markdownlint (0.37.0)

Spaces inside link text

(MD039, no-space-in-links)